.github/workflows/CI.yml

```diff
@@ -43,30 +60,49 @@ jobs:
   make_mason:
     runs-on: ubuntu-latest
     env:
+      # To try to keep this config from running the longest,
+      # use a simpler configuration
+      CHPL_TASKS: fifo
+      CHPL_MEM: cstdlib
+      CHPL_GMP: none
+      CHPL_RE2: bundled
+      CHPL_LLVM: none
+      # check that mason builds
       CHPL_SMOKE_SKIP_MAKE_MASON: false
+      # run make check in this config too
+      # to get more coverage for quickstart and because it's
+      # not the slowest config
+      CHPL_SMOKE_SKIP_MAKE_CHECK: false
     steps:
     - uses: actions/checkout@v2
+      #    - name: install deps
+      #run: sudo apt-get install $CHPL_APT_DEPS
     - name: make mason
       run: |
-        CHPL_LLVM=none ./util/buildRelease/smokeTest
+        ./util/buildRelease/smokeTest
 
-  check_annotations:
+  check_compiler_tests:
```

Comment:
Add a word to note this is next gen compiler, not current?

Reply:
Sure. `check_compiler_next_tests` ? `check_compiler_lib_tests` ? Something else?

Reply:
`check_compiler_next_tests` would be fine with me